ThunkLibs/GuestLibs/CMakeLists.txt

```diff
@@ -189,11 +190,7 @@ if (BITNESS EQUAL 64)
 
   find_package(PkgConfig)
   pkg_search_module(X11 REQUIRED x11)
-
-  string(REGEX MATCH "([0-9]*)\.([0-9]*)\.([0-9]*)" _ "${X11_VERSION}")
```

Comment:
In local testing, `string(REGEX MATCH "([0-9]*)\.([0-9]*)\.([0-9]*)" _ "1.10.430")` extracts the values "1", "10", and "430" fine for me. Is the problem somewhere else?

Reply:
With `1.13.1` it was pulling out 1.1.1 here, breaking the XCB version check I needed to add.

Reply:
Additional issue, one PC had version `1.14` which doesn't have a patch version, breaking the define generation as well.

Reply:
Found the difference between your output and mine. The regex breaks if their is no patch version in the version.
With the regex approach:
`1.14 -> 1.1.0`
`1.14.0 -> 1.14.0`

With the new approach:
`1.14 -> 1.14.0`
`1.14.0 -> 1.14.0`